pkgs/development/compilers/go/1.14.nix

```diff
@@ -11,8 +11,16 @@ let
 
   inherit (lib) optionals optionalString;
 
-  go_bootstrap = callPackage ./bootstrap.nix {
-    inherit Security;
+  go_bootstrap = callPackage ./binary.nix {
```

Comment:
You changed this to `binary.nix`, the others are `bootstrap.nix`.

Reply:
Good catch, amended.